### PR TITLE
Kill zombie server after running integration tests

### PR DESCRIPTION
**Pull request by jroebu14**
**Overall change:**
Stop the server from running after running integration tests.

**Code changes:**

- Added a `stop` npm script and changed all other scripts to use it for

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,9 +33,8 @@
     "test:puppeteer": "jest --ci --env=jsdom --colors ./puppeteer",
     "test:unit": "test -z $CI && jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\" || jest --ci --runInBand --env=jsdom --coverage --colors --testPathIgnorePatterns=\"src/integration\" \"puppeteer\"",
     "test:unit:watch": "npm run test:unit -- --watch",
-    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && jest --runInBand --colors ./src/integration/pages/**/*",
+    "test:integration": "./scripts/killApp.sh && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && (jest --runInBand --colors ./src/integration/pages/**/*) && ./scripts/killApp.sh",
```
Comment (karinathomasbbc) [inline package.json]: Could we add a new npm command and use this in place of killApp? e.g. `"stop": "./scripts/killApp.sh"`

This command then becomes:
```
"test:integration": "npm run stop && npm run build:local && (npm run start & wait-on -t 20000 http://localhost:7080/status) && (jest --runInBand --colors ./src/integration/pages/**/*) && npm run stop",
```

Comment (jroebu14) [reply package.json]: `stop:app` instead?

Comment (karinathomasbbc) [reply package.json]: Yeah, happy with that 👍

Comment (jroebu14) [reply package.json]: I went with `stop` as it's less characters to type and consistent with `start`

**Review by karinathomasbbc — APPROVED** [review]
Nice work, thanks for this 👍

**Review by andrewscfc — APPROVED** [review]
Thanks for sorting this

**Pull request merged**